partials/new_device_login.html

```diff
@@ -1,4 +1,8 @@
 <% extends "partials/base/base_text_only.html" %>
 
 <% block heading %>{{t "New sign-in to Firefox"}}<% endblock %>
-<% block text_primary %>{{{ device }}}<br/>{{ timestamp }}<% endblock %>
+<% block text_primary %>{{{ device }}}<br/>
+<span style="display:block;">{{t "IP address: %(ip)s" }}</span>
+{{ location }}
```

Comment:
@shane-tomlinson the div/p will result in different styles, which I will need to override again. Span will not cause that. Also, the reason for location to not be wrapped is to save the conditional check for existence of location, as otherwise an empty new line will be presented.